library/src/main/java/com/mux/stats/sdk/muxstats/MuxStateCollector.kt

```diff
@@ -360,6 +366,9 @@ open class MuxStateCollector(
   @Suppress("unused")
   fun incrementDroppedFrames(droppedFrames: Int) {
     this.droppedFrames += droppedFrames
+    muxStats.setDroppedFramesCount(this.droppedFrames.toLong())
+    // note - in the current implementation, we rely on a subsequent playback event to send this.
```

Comment:
Did we always rely on a subsequent playback event to send this?

Reply:
Separate question. Out of curiosity. How does this cross-cut with beacon flushing where only some playback events trigger a flush and some others do not?

Reply:
> Did we always rely on a subsequent playback event to send this?

Yeah as far as I know the ExoPlayer SDK worked the same way.

> Separate question. Out of curiosity. How does this cross-cut with beacon flushing where only some playback events trigger a flush and some others do not?

Theoretically if we had some kind of `droppedframes` event, it wouldn't have to be a `flushEvent`, so wouldn't necessarily cause beacons. We don't have an event for it though. We could send something, but i see no reason to rock the boat tbh